UniversityTimetablingSystem/src/University.cpp

```diff
@@ -263,47 +250,55 @@ std::vector<std::pair<Course, std::pair<TimeSlot, Instructor>>> University::sche
         newSchedule[randomIndex] = std::make_pair(newSchedule[randomIndex].first,
                                                   std::make_pair(newTimeSlot, newInstructor));
 
-        // calculate the cost of the new schedule
         double currentCost = calculateScheduleCost(currentSchedule);
         double newCost = calculateScheduleCost(newSchedule);
 
-        // decide whether to accept the new schedule(If the new schedule has a lower cost, it is
-        // always accepted,
-        //  otherwise the decision is made with a probability determined by the function
-        //  std::exp...)
         if (newCost < currentCost ||
             std::exp((currentCost - newCost) / temperature) > (double)std::rand() / RAND_MAX) {
             currentSchedule = newSchedule;
         }
 
-        // update the best schedule if the new schedule is better
         if (newCost < bestCost) {
             bestSchedule = newSchedule;
             bestCost = newCost;
         }
 
-        // cool down the temperature
```

Comment:
Comments that describe annealing algorithm and its parameters should remain. It will help you later to understand what does your algorithm